db/migrate/20171219091048_add_token_hash_to_invitation_token.rb

```diff
@@ -0,0 +1,7 @@
+class AddTokenHashToInvitationToken < ActiveRecord::Migration[5.0]
+  def change
+    add_column :invitation_tokens, :token_hash, :string
+    add_column :invitation_tokens, :expires_at, :datetime, null: false
```

Comment:
Gaat deze migratie errors geven als er al InvitationTokens bestaan in de database? Moet je niet eerst de property toevoegen en null toestaan, vervolgens alle InvitationToken objecten updaten, en daarna pas de null van die property op false zetten?

Reply:
Ja hier wou ik het nog met jou over hebben. Die invitation token objecten in de DB, die zouden we allemaal weg kunnen gooien toch? Als we dit mergen in bijv. het weekend. Dan zouden we verder geen migrations hoeven te maken.